app/models/recipe.rb

```diff
@@ -1,4 +1,6 @@
 class Recipe < ApplicationRecord
   belongs_to :recipe_type
-  has_many :item, through: :recipe_item
+  has_many :recipe_ingredients, dependent: :destroy
+
+  validates :recipe_ingredients, length: { minimum: 2 }
```

Comment:
Should not we add a maximum item too? At the moment, recipes can have a maximum of 5 items, right? We could just change this in the future if needed.

Reply:
Sure!